packages/node/src/cli.js

```diff
@@ -256,7 +256,20 @@ const main = async (args = hideBin(process.argv)) => {
         global: true,
         group: BEHAVIOR_GROUP,
       },
+      verbose: {
+        describe: 'Enable verbose logging',
+        type: 'boolean',
+        global: true,
+        group: BEHAVIOR_GROUP,
+      },
```

Comment:
```suggestion
      verbose: {
        alias: ['v'],
        describe: 'Enable verbose logging',
        type: 'boolean',
        global: true,
        group: BEHAVIOR_GROUP,
      },
```

Reply:
No, `-v` is not aliased to `--version` (nor is this yargs' default behavior)